extlibs/headers/catch.hpp

```diff
@@ -1,6 +1,6 @@
 /*
  *  Catch v1.12.2
```

Comment:
Have these fixes not been released yet?

Reply:
Nope. See: https://github.com/catchorg/Catch2/compare/v1.12.2...Catch1.x

Reply:
For reference, I ran the python2 script `generateSingleHeader.py` to generate the single header.